Description: fix test for nodejs >= 14
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1009600
Forwarded: no
Last-Update: 2022-04-13

--- a/safer-buffer/tests.js
+++ b/safer-buffer/tests.js
@@ -213,7 +213,7 @@
     t.throws(function () { impl.Buffer.alloc('b25ldHdvdGhyZWU=', 'base64') })
     t.throws(function () { impl.Buffer.alloc(-10) })
     t.throws(function () { impl.Buffer.alloc(1e90) })
-    t.throws(function () { impl.Buffer.alloc(2 * (1 << 30)) })
+    //t.throws(function () { impl.Buffer.alloc(2 * (1 << 30)) })
     t.throws(function () { impl.Buffer.alloc(Infinity) })
     t.throws(function () { impl.Buffer.alloc(-Infinity) })
     t.throws(function () { impl.Buffer.alloc(null) })
@@ -227,7 +227,7 @@
     t.throws(function () { dangerous.Buffer[method]('') })
     t.throws(function () { dangerous.Buffer[method]('string') })
     t.throws(function () { dangerous.Buffer[method]('string', 'utf-8') })
-    t.throws(function () { dangerous.Buffer[method](2 * (1 << 30)) })
+    //t.throws(function () { dangerous.Buffer[method](2 * (1 << 30)) })
     t.throws(function () { dangerous.Buffer[method](Infinity) })
     if (dangerous.Buffer[method] === buffer.Buffer.allocUnsafe) {
       t.skip('Skipping, older impl of allocUnsafe coerced negative sizes to 0')
